products/compute/helpers/terraform/disk_overrides.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+id: !ruby/object:Provider::Terraform::PropertyOverride
+  exclude: true
+sizeGb: !ruby/object:Provider::Terraform::PropertyOverride
+  name: size
+  default_from_api: true
+sourceSnapshot: !ruby/object:Provider::Terraform::PropertyOverride
+  name: snapshot
+  description: |
+    {{description}}* snapshot
```

Comment:
What does `{{description}} * snapshot` do?

Reply:
https://github.com/terraform-providers/terraform-provider-google/pull/1755/files#diff-a1e4f4837e83c9ac24d81e8adec94a89R149

(terraform is the only one that also accepts just the resource name)

Reply:
:man_facepalming: of course.